Makefile

```diff
@@ -107,3 +107,8 @@ dummy_translations: ## generate dummy translation (.po) files
 build_dummy_translations: extract_translations dummy_translations compile_translations ## generate and compile dummy translation files
 
 check_translations_up_to_date: build_dummy_translations detect_changed_source_translations ## validate translations
+
+test-python: clean ## run tests using pytest and generate coverage report
+	pytest
+
+test: test-python ## run tests
```

Comment:
@zubair-arbi this repo has Javascript tests so it should have `test-js` make target and then in `test` target we should invoke both python and js tests